pinot-query-runtime/src/test/resources/queries/JsonType.json

```diff
@@ -0,0 +1,22 @@
+{
+  "json_data": {
+    "tables": {
+      "jsonTbl": {
+        "schema": [
+          {"name": "jsonCol", "type": "JSON"},
+          {"name": "stringCol", "type": "STRING"}
+        ],
+        "inputs": [
+          ["{\"key1\":\"val1\",\"key2\":\"val2\"}", "str1"],
+          ["{\"key11\":\"val11\",\"key22\":\"val22\"}", "str22"],
+          ["{\"key111\":\"val111\",\"key222\":{\"key222_a\":\"val222_a\"}}", "str33"]
+        ]
+      }
+    },
+    "queries": [
+      {"sql": "SELECT jsonCol FROM {jsonTbl}"},
+      {"sql": "SELECT jsonCol, stringCol FROM {jsonTbl}"},
+      {"sql": "SELECT jsonCol, stringCol FROM {jsonTbl} where stringCol='str33'"}
```

Comment:
please also test the JSON functions and make sure those works

Reply:
I have a separate [issue](https://github.com/apache/pinot/issues/10767) for that. Would you like me to add support for those in this PR or a separate one?

Reply:
ah. i see so this is only to support JSON as string return? got it. that's good in its own.

Reply:
yes.